techniques/system/inventory/1.0/fusionAgent.st

```diff
@@ -142,14 +142,9 @@ bundle agent download_server_uuid
 {
   vars:
 
-    # If curl is available, use it
-    curl_installed::
+    # curl should always be available now
```

Comment:
could you also add a report_error report if curl_installed is not defined ? It would help debugging in the future